ansible/group_vars/develop/with.yml

```
@@ -3,4 +3,4 @@ WITH_SCHULCLOUD_INIT: true
 WITH_CALENDAR_INIT: true
 WITH_ERWINIDM: true
 WITH_OIDCMOCK: true
-WITH_H5P_EDITOR: false
+WITH_H5P_EDITOR: true
```

Set it back to the original value
```suggestion
WITH_H5P_EDITOR: false
```